force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -378,6 +572,16 @@ public with sharing class ClassesObjectsMethods {
      * @return The Person object of the older person.
      */
     public static Person getOlderPerson(Person person1, Person person2) {
-        return null; // Replace null with the variable you used to store the result
-    }
-}
\ No newline at end of file
+        if (person1.getAge() > person2.getAge())
+        {
+            return person1;
+        }
+        else if (person2.getAge() > person1.getAge())
+        // 
+            return person2;
+        }
+        else 
+        {
+            return person1;
```

Comment:
How could you use the `>=` operator in the first if statement to shorten up your code significantly?

Reply:
If person1.getAge() >= person2.getAge(){
    return person1;
}